queries.sql

```diff
@@ -8,3 +8,91 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > '10.5';
 SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN '10.4' AND '17.3';
+
+-- Start transaction 
+BEGIN;
+
+-- Update transaction
+UPDATE animals
+SET species = 'unspecified';
+
+SELECT * FROM animals;
+
+-- Return to previous state
+ROLLBACK;
+
+SELECT * FROM animals;
+
+-- Start transaction
+BEGIN;
+
+-- Update transaction
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
```

Comment:
- [ ] @MNisarAli great work on your query, kindly add a query to verify the changes were made before commit, also update the screenshots to showcase that.